### DISCUSSION TITLE
V1.3 bright vertical lines on the very left of the screen

### DISCUSSION THREAD

**Discussion by K9-711** [eduardofilo/RG350_adam_image#103] [Hardware problems?]
Those vertical lines are easy visible when screen is dark or bright. If screen bright, the lines tend to flickering. RG350M. 

Edit: Bright lines are suddenly gone.

**Comment by eduardofilo** [comment]
Can you put a picture of the screen showing the problem so I can compare with my screen?
It sounds to me a hardware problem in your unit. I have not noticed that in my machine, nor anyone has commented something similar.
In any case if there is a actual problem that produce that lines, surely it won't be in my hands to solve it. The image is just a normal installation of pieces of software maintained by other people.
But put a picture and I will give you my opinion.

**Comment by K9-711** [comment]
Hello. Thank you for your reply. Shortly after I posted the issue I edited
the post. The lines disappeared suddenly. Since then I rebooted several
times and all is fine.

BTW, I don't know if this is useful or not; after flashing 1.3 with Etcher
my windows 10 machine wasn't able to see the SD card. After deleting the
partitions and creating a new fat32 and flashing it again without success I
extracted the .xz file and flashed it with Win32 Disk Imager and windows 10
showed the SD card. Didn't have that issue with v1. 2.

However, the image is fantastic. Actually overwhelming :)

Thank you and have a great night.
Michael

**Comment by eduardofilo** [comment]
I'm glad there was nothing serious about the screen. Actually that screen although it looks great, it must be difficult to manage. With the stock firmware, I did very strange things when the GMenu2X screensaver was triggered that only resolved after several minutes with the console turned off.
I'm also glad that you like the image. Actually it is what I said before, a simple installation of system + frontend + emulators. In other words, I have not actually created anything.
Good night...
